ember-app/app/initializers/sockets.js

```diff
@@ -36,6 +37,7 @@ export function initialize(container, application){
         return callback;
       },
       unsubscribe: function(channel, callback) {
+        if(channel == "null") { return Ember.K; }
```

Comment:
null gets converted to a string somewhere upstream

It's the No Milestone column, we use this conventional string syntax and parser to bind to channel names and topics and it convert the repo name to "null" because of that column has no backing model. 

It's a one off edge case weird column that was kind of hacked together to get code reuse. 

I'll add a comment and another guard for actually null's for now, but this prevents bad subscription calls